install/kubernetes/helm/istio/charts/kiali/templates/secrets.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
```

Comment:
@gbaufake  One issue here, I think you need to mount the secret to kiali deployment?

Reply:
yeah, good catch.. @gbaufake you need this: https://github.com/kiali/kiali/blob/master/deploy/kubernetes/kiali.yaml#L78-L87

Reply:
Added on 847ab